src/components/ShareTooltip/ShareList/ShareList.tsx

```diff
@@ -97,10 +114,15 @@ export class ShareList extends React.PureComponent<ShareListInnerProps, ShareLis
     }
 
     private renderCopyLink() {
+        const {url, copyIcon, copyTitle, renderCopy} = this.props;
         const {copied} = this.state;
-        const label = copied ? i18n('label_copy-link-copied') : i18n('label_copy-link');
 
-        return (
+        const label =
+            copyTitle || (copied ? i18n('label_copy-link-copied') : i18n('label_copy-link'));
+
+        return renderCopy ? (
+            renderCopy({url, title: label, icon: copyIcon || Link, className: b('copy-link')})
```

Comment:
Let's create a wrapper with `b('copy-link')` around renderCopy and CopyToClipboard

Reply:
Yes, and not passing className to render function

Reply:
Done

Reply:
Seems styles are broken now, copy text is truncated

Reply:
I had to use the `React.cloneElement` function to extend readonly functional component with a custom class name. If you know more elegant  solution, please tell me.

Reply:
Unexpected behavior been in this code
```javascript
componentDidMount() {
        if (this.props.withCopyLink && this.copyLink) {
            this.copyLink.style.width = `${this.copyLink.scrollWidth}px`;
        }
    }
```
I have explicitly set `display` property to div wrapper container as at button